doc-coverage/raco.rkt

```diff
@@ -0,0 +1,75 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+(module+ main
+
+  (define binding (make-parameter #f))
+  (define ratio (make-parameter #f))
+  (define ignore (make-parameter #f))
+  (define error-on-exit? (make-parameter #f))
+
+  (define args
+    (command-line
+     #:program (short-program+command-name)
+     #:once-any
+     [("-b" "--binding") b
+      "Check the documentation for a specific binding"
+      (binding (string->symbol b))]
+     [("-r" "--ratio") r
+      "Specify required documentation ratio"
+      (ratio (string->number r))]
+     [("-s" "--skip") s
+      "Specify regex of bindings to ignore"
+      (ignore (regexp s))]
+     #:args (file . files)
+     (cons file files)))
+
+  (for ([a (in-list args)])
+    (let/ec break
+
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (set! a (string->symbol a)))])
+        (namespace-require a))
+      (with-handlers ([exn:fail? (lambda (e)
+                                   (fprintf (current-error-port) "Module ~a can not be loaded~n" a)
+                                   (error-on-exit? #t)
+                                   (break))])
```

Comment:
I guess `try-namespace-require` would be the name of the helper function from above. (I wouldn't put a `!` so that it matches the conventions of `namespace-require`, but if you're attached to having a `!` there I don't mind having it as well.)

I am a little more opposed to putting it in an `if` though. The body is already (imo) too indented. Having break continuation there (which I just realized should be called continue) is just the continue in other (usually C like) languages. And allows us to not needlessly further indent the function body. Although as this is your package, if that's the way you'd like it, I'd be willing to change it for you.